packages/mccs/mccs.1.1+10/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+maintainer: "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+authors: [
+  "Claude Michel <claude.michel@unice.fr>"
+  "Louis Gesbert <louis.gesbert@ocamlpro.com>"
+]
+homepage: "http://www.i3s.unice.fr/~cpjm/misc/"
+bug-reports: "https://github.com/AltGr/ocaml-mccs/issues"
+license: "LGPL-2.1 with OCaml linking exception, BSD-3-clause, GPL-3.0"
+dev-repo: "git+https://github.com/AltGr/ocaml-mccs.git"
+build: [
+  ["dune" "build" "-p" name]
+  ["sh" "-c" "dune build @settests --auto-promote || true"] {with-test}
+  ["dune" "runtest"] {with-test}
```

Comment:
```suggestion
  ["dune" "runtest" "-p" name "-j" jobs] {with-test}
```
Wouldn't it be better to use at least `-p` here?

Reply:
`--profile=release` is probably a good idea, yes. Not `-p`, though - or does that automatically select sub-packages too (there's a PR in the pipeline for ocaml-mccs to use virtual packages for the linking methods of the solver backend)